<launch>
    <node name="pylon_camera_node" pkg="pylon_camera" type="pylon_camera_node" output="screen" respawn="false" launch-prefix="">
        <rosparam command="load" file="$(find pylon_camera)/config/default.yaml" />
    </node>
    <node name="screw_detection" type="candidate_generator.py" pkg="screw_detection" output="screen"/>
    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="my_rqt_reconfigure" args="screw_detection"/>
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find screw_detection)/rviz_config.rviz" />
</launch>
